src/Tools/dotnet-dump/Commands/SOSCommand.cs

```diff
@@ -30,14 +30,15 @@ namespace Microsoft.Diagnostic.Tools.Dump
     [Command(Name = "name2ee",          AliasExpansion = "Name2EE",             Help = "Displays the MethodTable structure and EEClass structure for the specified type or method in the specified module.")]
     [Command(Name = "printexception",   AliasExpansion = "PrintException",      Help = "Displays and formats fields of any object derived from the Exception class at the specified address.")]
     [CommandAlias(Name = "pe")]
+    [Command(Name = "pathto",           AliasExpansion = "PathTo",              Help = "Displays the GC object path to the specified object.")]
```

Comment:
How does this differ from GCRoot?

Reply:
It is different from gcroot because it seems to take one object and display the path to another one.  I never did get it to work and it isn't documented anywhere. I was trying to be "complete" just in case some one out there wants to use it.  I'll remove it for now.